index.html

```diff
@@ -0,0 +1,39 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+<section class="wrap">  
+    
+
+<header class="headr">
+
+    <div class="flex">
+        <a href="#home" class="logo">My Logo</a>
+        <img src="unt/Union.png" alt="" class="bars">
+    </div>
+    
+</header>
+
+<section class="headline">
+
+        <h1>I’m John
+            Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#">LET'S CONNECT</a>
+        <img src="unt/social media icons.png" alt="">
```

Comment:
- [ ] I would kindly suggest the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 👍 👍 

> N.B: The social media icons should not be an image. it should be a list of the social media icons

E.g
```HTML
<ul class="social-icon">
     <li>
        <a href="#"><img src="./unt/Vector-3.svg" alt="twitter"></a>
     </li>
     <li>
        <a href="#"><img src="./unt/Linkedin icon.svg" alt="linked-in"></a>
     </li>
</ul>
```